### ISSUE TITLE
[BUG] possible oversights with latest guide update

### ISSUE THREAD

**Issue by MoerasGrizzly** [VivaNewVegas/vivanewvegas.github.io#40]
## Describe the bug

Consistent Pipboy Icons
https://vivanewvegas.github.io/hud.html#cpi
mentions the main file, but not the Vanilla UI Plus Patch. Oversight?
Mojave Raiders
https://vivanewvegas.github.io/content.html#raiders
has a "The LIving Desert" patch, but it's not part of the instructions. Oversight?

https://vivanewvegas.github.io/overhauls.html#unfoundloot_ini
Finally the Unfound Loot Custom INI was not updated since April, when the guide still used Fo3 Economy or Economy overhaul. This is mostly reflected in the "Max Item Value" Setting, which seems purposefully lowered compared to the Unfound Loot "Balanced" default because of the economy mods, which are now no longer present. Oversight?

**Comment by VivaNewVegas** [comment]
1). The VUI+ patch for CPI isn't needed with just the first main file, only with the second(?) main file that integrates the updated interfaceshared00.dds

2). Barring any other conflicts, the TLD patch shouldn't be needed since TLD disables the overpass settlement by default in its INI 

3). I will have to play the game some more with vanilla item values to get a better hold on how to update the custom INI. I think for now I may just remove the INI since iirc max item value was the only major change I made. I have also been thinking about removing unfound loot entirely and relying solely on JSUE's item rarity system, which I have made edits to in the new custom INI.
